loginFunctions.js

```diff
@@ -0,0 +1,103 @@
+const ERRORMESSAGE = {
+  wrongEmail: "잘못된 이메일 형식입니다",
+  wrongPassword: "비밀번호를 8자 이상 입력해주세요",
+  emailIsEmpty: "이메일을 입력해주세요",
+  passwordIsEmpty: "비밀번호를 입력해주세요",
+  nicknameIsEmpty: "닉네임을 입력해주세요",
+  passwordIsNotSame: "비밀번호가 일치하지 않습니다.",
+};
+
+const requireContent = (e) => {
+  console.log("requireContent called for:", e.target.id);
+  let content = e.target.value;
+  let emailRegex = /^[a-zA-Z0-9]+([._-][a-zA-Z0-9]+)*@[a-zA-Z0-9-]+(\.[a-zA-Z]{2,})+$/;
+
+  if (content) {
+    if (emailRegex.test(content) === false && e.target.id == "email") {
+      createNewMessage(ERRORMESSAGE.wrongEmail, e.target);
+    } else if (e.target.nextElementSibling?.tagName == "P") {
+      e.target.style = "border: none";
+      e.target.nextElementSibling.remove();
+      document.querySelector(".visible_icon").style = "bottom: 1.4rem;";
+    }
+    if (e.target.id == "password" && content.length < 8) {
+      createNewMessage(ERRORMESSAGE.wrongPassword, e.target);
+      document.querySelector(".visible_icon").style = "bottom: 6.9rem;";
+    }
+  } else {
+    let messageContent;
+    switch (e.target.id) {
+      case "email":
+        messageContent = ERRORMESSAGE.emailIsEmpty;
+        break;
+      case "password":
+        messageContent = ERRORMESSAGE.passwordIsEmpty;
+        break;
+      case "nickname":
+        messageContent = ERRORMESSAGE.nicknameIsEmpty;
+        break;
+    }
+    createNewMessage(messageContent, e.target);
+    document.querySelector(".visible_icon").style = "bottom: 6.9rem;";
+  }
+};
+
+const createNewMessage = (messageContent, target) => {
```

Comment:
💊 제안
에러 메시지를 보여주는 것이니 이를 알 수 있는 이름으로 하시는 것을 추천드립니다~

```suggestion
const createErrorMessage = (messageContent, target) => {
```